cluster-autoscaler/core/utils.go

```diff
@@ -152,6 +154,20 @@ func GetNodeInfosForGroups(nodes []*apiv1.Node, cloudProvider cloudprovider.Clou
 	return result, nil
 }
 
+func sanitizeTemplateNodeLables(node *apiv1.Node) *apiv1.Node {
+	obj, _ := api.Scheme.DeepCopy(node)
+	newNode := obj.(*apiv1.Node)
+	newNode.Labels = make(map[string]string, len(node.Labels))
+	for k, v := range node.Labels {
+		if k != "kubernetes.io/hostname" {
```

Comment:
Not a big deal, but I'd rather make it a const. Feel free to ignore this comment if you disagree.